litellm/llms/bedrock.py

```diff
@@ -702,6 +702,11 @@ def _embedding_func_single(
     encoding=None,
     logging_obj=None,
 ):
+    if type(input) != str:
```

Comment:
Heads up, I think this needs to be:

```
if isinstance(input, str) is False:
  ...
```

If you inherit from `str` then this rejects when I think it would be valid for it to work:

Example:

```
class CustomStr(str):
    pass

def test_lower_str():
    x = CustomStr("x")

    assert x == "x"
    assert x != "y"

    assert type(x) != str
    assert isinstance(x, str)
```

Reply:
updated !